pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1204,10 +1208,11 @@ func (m *kubeGenericRuntimeManager) computePodActions(ctx context.Context, pod *
 //  2. Kill pod sandbox if necessary.
 //  3. Kill any containers that should not be running.
 //  4. Create sandbox if necessary.
-//  5. Create ephemeral containers.
-//  6. Create init containers.
-//  7. Resize running containers (if InPlacePodVerticalScaling==true)
-//  8. Create normal containers.
+//  5. Invoke sandbox ready callback to Kubelet to update pod status
+//  6. Create ephemeral containers.
+//  7. Create init containers.
+//  8. Resize running containers (if InPlacePodVerticalScaling==true)
+//  9. Create normal containers.
```

Comment:
Hello @SergeyKanzhelev,

> What about DRA? Can we include it in the description as well. I wonder if there are more places where it should be mentioned that DRA allocation is also included into the time before this condition is set

I was preparing a separate PR to add tests that validates this order between the DRA allocate calls and PodReadytoStartContainers condition.

I can also extend the documentation comments over `SyncPod` function to clarify the above order.

Plus, I am also planning to update the Pod Lifecycle Docs to mention this.

Would that be sufficient? Or I'm missing some more places? Thanks!